src/rebar_prv_xref.erl

```diff
@@ -110,6 +131,17 @@ prepare(State) ->
                                 sets:from_list(ConfXrefChecks))),
     XrefChecks.
 
+lib_paths(AppInfo, State) ->
+    case rebar_state:get(State, xref_include_all_code, false) of
+        false ->
+            AppDetails = rebar_app_info:app_details(AppInfo),
+            Libs = proplists:get_value(applications, AppDetails, []) ++
+                proplists:get_value(included_applications, AppDetails, []),
+            [code:lib_dir(Dep, ebin) || Dep <- Libs];
```

Comment:
Not all application use the same directory name as the application name. Someone reported this on IRC the other day.